content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -559,14 +999,16 @@ For example, the default value of `createdAt` is [`now()`](../../../reference/ap
 
 ```prisma
 model Post {
-  id         Int         @id @default(autoincrement())
+  ?* ... ID field ... */
```

Comment:
?

Reply:
Changed to 2 tabs with mongodb and relational db example